frontend/src/components/navbar/navbar.css

```diff
@@ -0,0 +1,132 @@
+
+.logo {
+  height: 50%;
+  size: 50%;
+  margin-left: 20px;
+}
+
+/* .navbar-button:hover {
+  background-color: #B5B4D4;
+} */
+
+.navbar {
+  height: 6rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 0px;
+  background-color: #793AFC;
+  text-transform: lowercase;
+}
+
+.navbar-button {
+    background-color: #6B6AAA;
+    font-size: 2rem;
+    color: #FBFAFB;
+    font-family: 'Nunito', sans-serif;
+    font-weight: 900;
+    margin-right: 20px;
+    box-shadow: 5px 5px black;
+    /* cursor: pointer; */
+    transition: 0.2s;
+    border-radius: .5rem;
+}
+
+/* Navbar when user is logged in */ 
+
+.welcome-header {
+  font-size: 2rem;
+  color: #fff; 
+  font-family: 'Nunito', sans-serif;
+  font-weight: 900;
+  margin-right: 20px;
+}
+
+.navbar label, #hamburger { display: none; }
+
+
+@media screen and (max-width: 768px) {
+    .navbar label {
+        display: inline-block;
+        color: white;
+        font-style: normal;
+        font-size: 1.2em;
+        padding: 10px;
+        /* cursor: pointer; */
+    }
+
+
+    .welcome-header {
+        display: none;
+    }
+
+    #hamitems { 
+        display: none;
+    }
+
+    #hamitems button {
+    box-sizing: border-box;
+    display: block;
+    width: 100%;
+    border-top: 1px solid #333;
+    }
+
+    .navbar input:checked ~ #hamitems {
+         display: block;
+        }
+
+    .navbar-buttons-wrapper {
+        position: absolute;
+        top: 25px;
+        right: 1px; 
+    }
+}
+
+
+/* Navbar when user is not logged in */ 
+
+.navbar label, #hamburger { display: none; }
+
+#hamitems { 
+    display: flex; 
+    position: relative;
+    z-index: 100;
+}
+
+.demo-button {
+  background-color: coral;
+}
+
+
+@media screen and (max-width: 768px) {
+    .navbar label {
+        display: inline-block;
+        color: white;
+        font-style: normal;
+        font-size: 1.2em;
+        padding: 10px;
+        /* cursor: pointer; */
+    }
+
+    #hamitems { 
+        display: none;
+    }
+
+    /* #hamitems button {
+    box-sizing: border-box;
+    display: block;
+    width: 100%;
+    border-top: 1px solid #333;
+    } */
+
+    /* .navbar input:checked ~ #hamitems { display: block; } */
+
+    /* .navbar-buttons-wrapper {
+    position: absolute;
+    top: 25px;
+    right: 1px; 
+    } */
```

Comment:
Also @bqh5026 Why are these commented out lines in here? It would be better if we didn't have them.